com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ModelBuilder.java

```diff
@@ -60,6 +60,9 @@ public Task<Void> create(IResource input) throws IOException, CompileExceptionEr
             taskBuilder.addInput(animations);
         }
         for (String t : modelDescBuilder.getTexturesList()) {
+            if (t.isEmpty())
+                continue; // TODO: Perhaps we can check if the material expects textures?
```

Comment:
It wasn't possible to build a model with no textures.

Reply:
We can check if there's samplers in the material shaders, but I don't think it should be a build error. In the case of the PBR shaders for instance, we want to be able to set global textures for all models in runtime